src/util/sss_regexp.c

```diff
@@ -0,0 +1,144 @@
+/*
+    SSSD
+
+    Authors:
+        Tomas Halman <thalman@redhat.com>
+
+    Copyright (C) 2018 Red Hat
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "util/sss_regexp.h"
+#include <string.h>
+
+#define SSS_REGEXP_OVEC_SIZE 30
+#define SSS_REGEXP_ERR_MSG_SIZE 120 // 120 is recomended by pcre2 doc
```

Comment:
Our code style policy says that we prefer the traditional comments (`/*...*/`) over the "C++" style. It's better to be consistent across our codebase.